legacyconfig/v1/types.go

```diff
@@ -413,7 +413,7 @@ type ImagePolicyConfig struct {
 	// registry. The external hostname should be set only when the image registry
 	// is exposed externally. The value is used in 'publicDockerImageRepository'
 	// field in ImageStreams. The value must be in "hostname[:port]" format.
-	ExternalRegistryHostname string `json:"externalRegistryHostname,omitempty"`
+	ExternalRegistryHostnames []string `json:"externalRegistryHostnames,omitempty"`
```

Comment:
this one should not change.  This format is used to read existing masterconfigs.